cmd/controller-manager/app/controller-manager.go

```diff
@@ -149,7 +149,7 @@ func Run(opts *options.Options, stopChan <-chan struct{}) error {
 
 	if opts.Scope == apiextv1.NamespaceScoped {
 		opts.Config.TargetNamespace = opts.Config.KubeFedNamespace
-		klog.Infof("KubeFed will be limited to the %q namespace", opts.Config.KubeFedNamespace)
+		klog.InfoS("KubeFed will be limited to the namespace", opts.Config.KubeFedNamespace)
```

Comment:
this is missing a key:
```suggestion
		klog.InfoS("KubeFed will be limited to the namespace", "namespace", opts.Config.KubeFedNamespace)
```